client/src/components/history/history-chat.tsx

```diff
@@ -87,13 +87,28 @@ export function HistoryChat(args: ScrollingQuestionsParams): JSX.Element {
 
   const dispatch = useDispatch();
   const [checked, toggleChecked] = useState<boolean>(true);
+  // const [idxQuestion, setIdxQuestion] = useState<number>(-1);
+
+  interface hashMap {
```

Comment:
naming style in typescript for anything "type like" (an interface, a class, or a type) is universally Pascal case, e.g. in this case it would be `HashMap`

...but this is a fully extraneous type anyway. Use the built in typescript type `Record<KeyType, ValueType>`, so in this case would be `Record<number, boolean>`